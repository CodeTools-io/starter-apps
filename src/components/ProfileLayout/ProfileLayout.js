import React from 'react';
import { Box } from 'grommet';

export default function ProfileLayout({ children }) {
  return (
    <Box className="ProfileLayout" fill>
      {children}
    </Box>
  );
}
