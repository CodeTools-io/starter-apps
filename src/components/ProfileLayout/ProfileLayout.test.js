import React from 'react'
import ProfileLayout from './ProfileLayout'

describe('ProfileLayout', () => {
  test('can render', () => {
    const { container } = renderComponent(
      <ProfileLayout>
        <p>Content for the ProfileLayout</p>
      </ProfileLayout>
    )

    expect(container).toMatchSnapshot()
  })
})

