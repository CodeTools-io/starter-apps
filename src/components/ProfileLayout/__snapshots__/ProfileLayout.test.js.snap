// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileLayout can render 1`] = `
<div>
  <div
    class="StyledBox-sc-13pk1d4-0 iNKrsm ProfileLayout"
  >
    <p>
      Content for the ProfileLayout
    </p>
  </div>
</div>
`;
